index.html

```diff
@@ -54,5 +56,249 @@ <h1 class="name">
+                <h4 class="counter-shadow">2015</h4>
+              </div>
+          <!--  -->
+    
+          <div class="project-details">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <!--  -->
+    
+          <div>
+            <ul class="tech-Stack">
+              <li id="html">HTML</li>
+              <li id="css">CSS</li>
+              <li id="js">JAVASCRIPT</li>
+            </ul>
+          </div>
+    
+          <!--  -->
+          <div><button type="button" class="seeProject">see project</button></div>
+    
+        </div>
+    </div>
+</section>
+
+<!-- about me section -->
+
+<section class="about-me">
+
+    <div class="myself">
+        <div class="">
+          <h2 class="headSelf">
+            About Myself
+          </h2>
+
+          <p class="text">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <p class="connect">
+            LET'S CONNECT
+           </p>
+           
+           <div class="social-media">
+               <ul> 
+                   <li><a href="" id="twitter"><img src="images/twitter.png" alt=""></a> </li>
+                   <li><a href="" id="linkin" ><img src="images/Linkedin icon.png" alt=""></a> </li> 
+                   <li><a href="" id="medium"><img src="images/medium.png" alt=""></a> </li>
+                   <li><a href="" id="github"><img src="images/github.png" alt=""></a></li>
+                   <li><a href="" id="angelist"><img src="images/angellist.png" alt=""></a></li>
+               </ul>
+           </div>
+          <p><button type="button" class="about2">Get my resume</button></p>
+        </div>
+<!--  -->
+        <ul class="languages">
+          <li class="">
+            <div class="arrow">
+              <h3 class="lan">Languages</h3>
+              <img class="arrowsign" src="images/Union.png" alt="" />
+            </div>
+
+            <div class="">
+              <img src="/images/Frame 9.png" alt="image" />
+              <img src="/images/Frame 10.png" alt="image" />
+              <img src="/images/Frame 11.png" alt="image" />
+            </div>
+          </li>
+          <li class="arrow">
+            <h3 class="lan">FRAMEWORKS</h3>
+            <p><img class="election2" src="./images/./left.png" alt="" /></p>
+          </li>
+          <li>
+            <img
+              class="separator"
+              src="images/Separator Bottom.png"
+              alt=""
+            />
+          </li>
+
+          <li class="arrow">
+            <h3 class="lan">Skills</h3>
+            <div>
+              <img class="election2" src="./images/./left.png" alt="" />
+            </div>

```

Comment:
- [ ] Please note that for orderliness and semantic purpose, you are advised to nest the ul so it will be like

```
<ul>
  <li>
    <ul>
      <li>
        <h3 class="lan">Languages</h3>
           <img class="arrowsign" src="images/Union.png" alt="" />
        </li>
        <li class="skill">JavaScript</li>
            <li class="skill">Ruby</li>
            <li class="skill">HTML</li>
            <li class="skill">CSS</li>
    </ul>
  </li>

Another li then ul inside it for the second one.
</ul>
```